src/StaticArrays.jl

```diff
@@ -50,7 +50,7 @@ include("inv.jl")
 include("eigen.jl")
 include("cholesky.jl")
 
-include("FixedSizeArrays.jl")
+# include("FixedSizeArrays.jl")
```

Comment:
@SimonDanisch, what is your opinion about this?

There seems to be some modifications to `FixedSizeArrays.jl`. Would some more modification "rescue" this feature? 

Do many people use the FixedSizeArrays feature?